project/frontend/src/components/PreferenceForm.css

```diff
@@ -0,0 +1,77 @@
+.form-container {
+  max-width: 800px;
+  text-align: left;
+  margin: 0px auto;
+}
+
+.form-container button {
+  display: block;
+  margin: 16px auto;
+}
+
+.form {
+  margin-bottom: 0px;
+}
```

Comment:
form, form-container, and header are also a bit generic since we have a lot of forms in this app. I think `preference-form-...` would be better.